golang/kurtosis-lambda/Dockerfile

```diff
@@ -9,17 +9,17 @@ ENV CGO_ENABLED=0
 
 WORKDIR /build
 # Copy and download dependencies using go mod
-COPY go.mod .
-COPY go.sum .
+COPY ../go.mod .
```

Comment:
Why is this different from Kurt Libs? I'd guess this has to do with the Docker build context being different

Reply:
When I was trying to fix the error to build the Docker image, one of the things that I was testing was changing these values, but then when I could build the image I forgot to return these commands to their original values.
So, good catch!